public/admin.js

```diff
@@ -1,2 +1,43 @@
-
+async function updateBook() {
+  let response = await fetch("http://localhost:3001/updateBook", {
+    method: "PATCH",
+    headers: { "Content-Type": "application/json" },
+    body: JSON.stringify({
+      id: 3,
+      title: "Legends of Arathrae",
+    }),
+  });
+  let updatedBook = await response.json();
+  console.log(updatedBook);
```

Comment:
very small nit: If you get the chance to remove 'console.logs' (that aren't intended for the user to actually see) before submitting that's always good practice :D